bin/varnishd/cache/cache_acceptor.c

```diff
@@ -348,10 +348,10 @@ vca_mk_uds(struct wrk_accept *wa, struct sess *sp, char *laddr, char *lport,
 	SES_Set_String_Attr(sp, SA_CLIENT_IP, "0.0.0.0");
 	SES_Set_String_Attr(sp, SA_CLIENT_PORT, "0");
 
-	strcpy(laddr, "0.0.0.0");
-	strcpy(raddr, "0.0.0.0");
-	strcpy(lport, "0");
-	strcpy(rport, "0");
+	bstrcpy(laddr, "0.0.0.0");
+	bstrcpy(raddr, "0.0.0.0");
+	bstrcpy(lport, "0");
+	bstrcpy(rport, "0");
```

Comment:
Like I said during bugwash you also need a replacement for `strncpy` because not all destination arguments are static buffers, so you need to be able to put the destination yourself and let the abstraction assert.

Reply:
/me bad. I should not have changed this case into bstrcpy.

Reply:
I'm pretty sure it's not the only case, I think there were others when I had a first look on Friday.